packages/agent/src/agent/http/types.ts

```diff
@@ -109,8 +110,9 @@ export function makeNonce(): Nonce {
   const buffer = new ArrayBuffer(16);
   const view = new DataView(buffer);
   const now = BigInt(+Date.now());
-  const randHi = Math.floor(Math.random() * 0xffffffff);
-  const randLo = Math.floor(Math.random() * 0xffffffff);
+  const randHi = Math.floor(randomNumber() * 0xffffffff); //?
+  const randLo = Math.floor(randomNumber() * 0xffffffff); //?
```

Comment:
Suggestion: make it so we don't (in several cases) divide by 4294967295 only to multiply by 0xffffffff here
```suggestion
  const randHi = Math.floor(randomNumber()); //?
  const randLo = Math.floor(randomNumber()); //?
```